apps/fz_http/lib/fz_http/users/user.ex

```diff
@@ -35,6 +39,7 @@ defmodule FzHttp.Users.User do
 
     has_many :devices, Device, on_delete: :delete_all
     has_many :oidc_connections, Connection, on_delete: :delete_all
+    has_many :api_tokens, ApiToken, on_delete: :delete_all
```

Comment:
@akrousset Doing the cascade delete here.

Reply:
@akrousset Actually, hmmm -- looks like this may be incorrect and [needs cascade delete on the table too](https://elixirforum.com/t/delete-nested-associations/7968/4).

Reply:
Yeah I saw that when doing my initial review, but quickly hit the `Viewed` checkbox and forgot about it by the time I got to the migration file. Woops!

Reply:
Oh, hmm, yeah, sounds like the migration will need it.